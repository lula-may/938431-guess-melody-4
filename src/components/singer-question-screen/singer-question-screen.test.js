import React from "react";
import renderer from "react-test-renderer";
import SingerQuestionScreen from "./singer-question-screen.jsx";

it(`Should render SingerQuestionScreen`, () => {
  const tree = renderer.create(
      <SingerQuestionScreen/>
  )
  .toJSON();

  expect(tree).toMatchSnapshot();
});

