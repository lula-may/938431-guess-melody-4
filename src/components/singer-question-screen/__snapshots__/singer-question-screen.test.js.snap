// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render SingerQuestionScreen 1`] = `
<section
  className="game game--artist"
>
  <header
    className="game__header"
  >
    <a
      className="game__back"
      href="#"
    >
      <span
        className="visually-hidden"
      >
        Сыграть ещё раз
      </span>
      <img
        alt="Угадай мелодию"
        className="game__logo"
        src="img/melody-logo-ginger.png"
      />
    </a>
    <svg
      className="timer"
      viewBox="0 0 780 780"
      xmlns="http://www.w3.org/2000/svg"
    >
      <circle
        className="timer__line"
        cx="390"
        cy="390"
        r="370"
        style={
          Object {
            "filter": "url(#blur)",
            "transform": "rotate(-90deg) scaleY(-1)",
            "transformOrigin": "center",
          }
        }
      />
    </svg>
    <div
      className="game__mistakes"
    >
      <div
        className="wrong"
      />
      <div
        className="wrong"
      />
      <div
        className="wrong"
      />
    </div>
  </header>
  <section
    className="game__screen"
  >
    <h2
      className="game__title"
    >
      Кто исполняет эту песню?
    </h2>
    <div
      className="game__track"
    >
      <div
        className="track"
      >
        <button
          className="track__button track__button--play"
          type="button"
        />
        <div
          className="track__status"
        >
          <audio
            src="https://music/Lady_Gaga.ogg"
          />
        </div>
      </div>
    </div>
    <form
      className="game__artist"
    >
      <div
        className="artist"
      >
        <input
          className="artist__input visually-hidden"
          id="artist-0"
          name="answer"
          onChange={[Function]}
          type="radio"
          value="artist-0"
        />
        <label
          className="artist__name"
          htmlFor="artist-0"
        >
          <img
            alt="Lady Gaga"
            className="artist__picture"
            src="https://images/Lady_Gaga.jpg"
          />
          Lady Gaga
        </label>
      </div>
      <div
        className="artist"
      >
        <input
          className="artist__input visually-hidden"
          id="artist-1"
          name="answer"
          onChange={[Function]}
          type="radio"
          value="artist-1"
        />
        <label
          className="artist__name"
          htmlFor="artist-1"
        >
          <img
            alt="Witney Houston"
            className="artist__picture"
            src="https://images/Witnew_Houston.jpg"
          />
          Witney Houston
        </label>
      </div>
      <div
        className="artist"
      >
        <input
          className="artist__input visually-hidden"
          id="artist-2"
          name="answer"
          onChange={[Function]}
          type="radio"
          value="artist-2"
        />
        <label
          className="artist__name"
          htmlFor="artist-2"
        >
          <img
            alt="Cher"
            className="artist__picture"
            src="https://images/Cher.jpg"
          />
          Cher
        </label>
      </div>
    </form>
  </section>
</section>
`;
